### ISSUE TITLE
Right click Package as Executable not working

### ISSUE THREAD

**Issue by adamdriscoll** [adamdriscoll/poshprotools#61]
As of version 1.0.258, Package as Executable no longer works. 

> I have been using the paid version for a few months now and I absolutely love it.  With the latest release I noticed that when I right click and select "Package as executable" nothing happen.  No errors, no changes, nothing.

**Comment by adamdriscoll** [comment]
Working fine with current version.

**Comment by stalliondan** [comment]
Hello,

I realized what the issue is.  I have dictated in the package settings that the Form.ps1 be the entry point of the script since the main code for the script is in this file.  When setting the Form.ps1 as the entry point nothing happens when attempting to package as .exe.  The strange part is that I do have a Form.exe in the folder dated back to July which indicates to me that at one time this worked.  Any help would be greatly appreciated.
